examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>Conventional case? Conventional to whom?</p><p> </p><p>"Note that if one assumes non-constant material parameters when deriving the higher time derivatives, it will be necessary..." - reference or brief explanation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>